mobile/src/main/java/org/openhab/habdroid/ui/ConnectionWebViewClient.kt

```diff
@@ -62,6 +71,36 @@ open class ConnectionWebViewClient(
         }
     }
 
+    override fun onReceivedClientCertRequest(view: WebView, request: ClientCertRequest) {
+        Log.d(TAG, "SSL Cient Cert required")
+        val prefs = view.context.getPrefs()
+        if (prefs.isDemoModeEnabled()) {
+            request.cancel()
+            return
+        }
+
+        val alias = prefs.getString(Constants.PREFERENCE_SSL_CLIENT_CERT, null)
+        Log.d(TAG, "Using alias " + alias)
+        if (alias == null) {
+            request.cancel()
+            return
+        }
+
+        GlobalScope.launch(Dispatchers.IO) {
+            try {
+                val chain = KeyChain.getCertificateChain(view.context, alias)
+                val privateKey = KeyChain.getPrivateKey(view.context, alias)
+                request.proceed(privateKey, chain)
+            } catch (e: KeyChainException) {
+                Log.d(TAG, "KeyChain Exception", e)
```

Comment:
```suggestion
                Log.d(TAG, "Error getting chain or private key", e)
```

The exception itself is also passed the log line, so the string doesn't need to contain the exception name.